### PR TITLE
Add RDS instance creation instructions to README

### PR DESCRIPTION
**Pull request by rrrkharse**
**Which issue is resolved by this Pull Request:**
Moving RDS instance creation instructions to the RDS setup README to consolidate pipelines/katib manifest configuration instructions within this repo. 

**Description of your changes:**
- Adapted the instructions from https://www.kubeflow.org/docs/distributions/aws/customizing-aws/rds/ to fit the README format of `distributions/aws/examples/rds-s3/README.md`.
- Added a sample `rds.yaml` template with the option to customize MySQL engine version as a configurable parameter
- Copied relevant sample images in the instructions
  - https://github.com/kubeflow/website/blob/master/content/en/docs/images/aws/cloudformation-rds-output.png

**Testing:**
Performed a manual setup of rds and s3 using the added instructions. Performed verification steps under https://github.com/awslabs/kubeflow-manifests/tree/v1.3-branch/distributions/aws/examples/rds-s3#40-verify-the-installation

### CODE REVIEW

File: distributions/aws/examples/rds-s3/README.md [hunk 1]
```diff
@@ -97,40 +97,55 @@ aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
 
 2. Create RDS Instance
 
-Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aws/rds/#deploy-amazon-rds-mysql) to set up an AWS RDS instance. Please follow only section called `Deploy Amazon RDS MySQL`. This RDS Instance will be used by Pipelines and Katib.
+Amazon RDS is a managed service that makes it easy to set up, operate, and scale a relational database in the AWS Cloud. It provides cost-efficient, resizable capacity for an industry-standard relational database and manages common database administration tasks. This RDS Instance will be used by Pipelines and Katib.
 
-To use the latest MySQL version (or a different version) in the CFN template, change the `EngineVersion` property of resource `MyDB` to desired version.
+The folloing instructions will show how to setup an RDS instance to be used by the Kubeflow Pipelines and Katib components.
```
Comment (ryansteakley) [inline distributions/aws/examples/rds-s3/README.md:102]: nit: folloing -> following

File: distributions/aws/examples/rds-s3/README.md [hunk 2]
```diff
@@ -97,40 +97,55 @@ aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
 
 2. Create RDS Instance
 
-Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aws/rds/#deploy-amazon-rds-mysql) to set up an AWS RDS instance. Please follow only section called `Deploy Amazon RDS MySQL`. This RDS Instance will be used by Pipelines and Katib.
+Amazon RDS is a managed service that makes it easy to set up, operate, and scale a relational database in the AWS Cloud. It provides cost-efficient, resizable capacity for an industry-standard relational database and manages common database administration tasks. This RDS Instance will be used by Pipelines and Katib.
```
Comment (surajkota) [inline distributions/aws/examples/rds-s3/README.md:100]: shall we move this intro to beginning of section 2.0

File: distributions/aws/examples/rds-s3/README.md (hunk 1 above)
Comment (surajkota) [reply distributions/aws/examples/rds-s3/README.md:102]: duplicate line. Already mentioned in previous para

File: distributions/aws/examples/rds-s3/README.md [hunk 3]
```diff
@@ -97,40 +97,55 @@ aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
...
 
-To use the latest MySQL version (or a different version) in the CFN template, change the `EngineVersion` property of resource `MyDB` to desired version.
+The folloing instructions will show how to setup an RDS instance to be used by the Kubeflow Pipelines and Katib components.
 
-For example:
+To get started deploying a MySQL database using Amazon RDS, you'll need to retrieve some configuration parameters that are needed.
 
+```shell
+export CLUSTER_NAME=<YOUR_CLUSTER_NAME>
```
Comment (surajkota) [inline distributions/aws/examples/rds-s3/README.md:107]: var already exported in previous sections

File: distributions/aws/examples/rds-s3/README.md [hunk 4]
```diff
@@ -97,40 +97,55 @@ aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
...
+
+# Retrieve your VpcId
+aws ec2 describe-vpcs \
```
Comment (surajkota) [inline distributions/aws/examples/rds-s3/README.md:110]: add --region in all these commands?

File: distributions/aws/examples/rds-s3/README.md [hunk 5]
```diff
@@ -97,40 +97,55 @@ aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
...
-    VPCSecurityGroups:
-      Ref: SecurityGroupId
-  DeletionPolicy: Snapshot
-
-  ...
-```
+
+With this information in hand, you can now use either the Amazon RDS console or use the attached [CloudFormation template](/distributions/aws/examples/files/rds.yaml) to deploy your database.
+
+:warning: The CloudFormation template deploys Amazon RDS for MySQL that is intended for Dev/Test environment.
```
Comment (surajkota) [inline distributions/aws/examples/rds-s3/README.md:133]: after reading section 2.0, I am wondering why are we maintaining 2 versions i.e. manual and automated setup. Lets add these warnings and instructions to retrieve etc to the automated section above and just maintain just that one

this is using cloudformation and so is automated as well

Let me know your thoughts

**Review by surajkota — CHANGES_REQUESTED** [review]

File: distributions/aws/examples/rds-s3/README.md (hunk 5 above)
Comment (goswamig) [reply distributions/aws/examples/rds-s3/README.md:133]: I also noticed we have PR https://github.com/awslabs/kubeflow-manifests/pull/107/files which creates RDS etc. So why do we need this or other? 

do customer care about CFN vs python script ?

**Comment by rrrkharse** [comment]
I agree with the points above that having two automated setup options is confusing and not convenient to maintain. I am removing the CFN installation steps as the automated python script is more streamlined (better automation of getting required parameters for creating RDS, etc.). I also feel like that a purpose and motive for using RDS and S3 are missing for those looking to scale their KFP and Katib installations. I will create a new PR targeting those changes.

**Comment by rrrkharse** [comment]
New PR here: https://github.com/awslabs/kubeflow-manifests/pull/113

**Pull request closed without merging**